pom.xml

```diff
@@ -722,6 +722,12 @@
               <groupId>org.mockito</groupId>
               <artifactId>mockito-core</artifactId>
               <version>${mockito.version}</version>
+              <exclusions>

```

Comment:
Hi @arturobernalg!

please also revert commit 354ba9cfa5cf9a5e7b8e776b17df87a780540cd0 and apply it here, so we can test with the new plugin that:
* it still performs its tasks (files are minified, merged etc.)
* mvn dependency:go-offline ends ok (it was the part failing on the Docker build)

Reply:
mmm 
With version 1.9.0, there's no need to exclude any libraries. I'll continue working on fixing the build with version 1.10.1.